feature/platform/transceiver/zr_inventory_test/README.md

```diff
@@ -0,0 +1,62 @@
+# TRANSCEIVER-7: Telemetry: 400ZR module inventory information. 
+
+## Summary
+
+Validate 400ZR modules report inventory information part number and serial
+number.
+
+## Procedure
+
+*   Plug in the ZR module in the host port and make sure the transceiver 
+    state is enabled and host is able to detect the module.
+*   With the module recognized verify it reports correct inventory
+    information by subscribing ON_CHANGE to the following telemetry paths.
+
+    *   /platform/components/component/state/serial-no
+    *   /platform/components/component/state/part-no
+
+*   Validate the streamed inventory information data is of type String.
+
+*   Verify that the modules inventory information is reported correctly after
+    an optic software reset.
+
+    *   With ZR module plugged in the host and properly recognized 
+    *   Verify the ZR optics inventory is correctly reported via the 
+        streaming telemetry paths above.
+    *   Reset the optic by enabling and disabling the transceiver state
+        through /components/component/transceiver/config/enabled.
+    *   Wait atleast 20 seconds in between toggling transceiver state.
+    *   Verify the ZR optics still reports correct inventory information.
+    *   Telemetry subscription should be ON_CHANGE and streamed data should
+        be of type String.
+
+*   Verify that the modules inventory information is reported correctly when
+    interface state is disabled.
+
+    *   With ZR module plugged in the host and properly recognized
+    *   Use /interfaces/interface/config/enabled to disable the module
+        interface state, wait 20 seconds. 
+    *   Verify the ZR optics inventory information is correctly reported via
+        the streaming telemetry paths above in this state.
+    *   Telemetry subscription should be ON_CHANGE and streamed data should
+        be of type String.
+
+*   Verify the module behaviour when transceiver state is set to disabled.
+
+    *   With ZR module plugged in the host and properly recognized.
+    *   Use /components/component/transceiver/config/enabled to disable the
+        module transceiver state, wait 20 seconds. 
+    *   Verify the ZR module is powered off and no inventory information
+        reported via the streaming telemetry paths above in this state.
+    *   Telemetry subscription should be ON_CHANGE and there should be no
+        streamed inventory data in this state.
+
```

Comment:
Can we also please add these test cases?

1. Verify the module behaviour when a transceiver is swapped with a different one.
2. Verify the module behaviour when a transceiver is  physically removed.

We have encountered both of these situations before and generally boil down to the device not correctly streaming deletes, but it would be good to explicitly call them out.

Reply:
Done

Reply:
Sorry to contradict this, but we don't expect to have the hardware/robotics available to automate these kinds of actions.  The README for tests in featureprofiles should only describe tests that are intended to be automated.

Reply:
Actually this test case should be already covered when we power off the transceiver, so that is equivalent to physically reseating the module. Ill delete this test here.

Reply:
Ah, gotcha thanks for the clarification.